modules/functional-test/src/main/java/com/intuit/wasabi/tests/library/util/Constants.java

```diff
@@ -106,4 +106,26 @@
 
     public static final String ROLE_LIST = "roleList";
 
+    /**
+     * This enum class holds the timeZones of various regions..added a few timezones
+     * feel free to add your timeZone if it is not there...
+     */
+    public static  enum TimeZoneEnum {
```

Comment:
this one will be used for the card view API @bargenilesh where we send the offset as part of the request